inventory/main/src/main/java/org/opennms/horizon/inventory/grpc/NodeGrpcService.java

```diff
@@ -132,6 +133,18 @@ public void createNode(NodeCreateDTO request, StreamObserver<NodeDTO> responseOb
         }
     }
 
+    @Override
+    public void updateNode(NodeUpdateDTO request, StreamObserver<Int64Value> responseObserver) {
+        try {
+            String tenantId = tenantLookup.lookupTenantId(Context.current()).orElseThrow();
+            var nodeId = nodeService.updateNode(request, tenantId);
```

Comment:
Nit: I tend to prefer the explicit type if the assigned type isn't obvious from the statement. In this case I was not sure if it's a `Long` or a `String`

```suggestion
            Long nodeId = nodeService.updateNode(request, tenantId);
```